elastic/kibana
# [Infra] Fix metric table fields and calculations

## Summary

Fixes #133119 (Uptime metric reporting)
Fixes #133122 (Wrong field for CPU metric)
Fixes #133124 (Percentage values are unscaled)

Hosts:
<img width="1412" alt="Screenshot 2022-06-17 at 21 19 12" src="https://user-images.githubusercontent.com/2564140/174388959-c9fc5ea5-26ff-45e1-99db-d250e3b695f4.png">

Pods:
<img width="1418" alt="Screenshot 2022-06-17 at 21 19 24" src="https://user-images.githubusercontent.com/2564140/174388964-db6e2aca-666f-4947-9e28-877f828e5639.png">

Containers:
<img width="1423" alt="Screenshot 2022-06-17 at 21 19 32" src="https://user-images.githubusercontent.com/2564140/174388966-0ec52321-9972-41f9-9131-f2341c569281.png">

### How to test

- Get some `system` and `kubernetes` module data (easily done by using the Edge cluster as a remote via the OBLT CLI)
- Configure the Metrics UI to read data from the remote clusters `metrics-*` indices
- Enable the Infrastructure tab technical preview with `observability:enableInfrastructureView`
- Go to the Infrastructure tab for any APM service, should see sub tabs for Hosts, Pods and Containers that show each table type
- Verify that the values look reasonable (uptime isn't based on epoch start, percentage values are not too low, values match up with what is shown in Inventory view or Node Details page, roughly, no empty values)
